website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,744 @@
+                          </div>
+                          <div className="min-w-0 flex-1">
+                            <h3
+                              onClick={() => handleExtensionClick(extension)}
+                              onKeyDown={e => {
+                                if (e.key === 'Enter' || e.key === ' ') {
+                                  e.preventDefault();
+                                  handleExtensionClick(extension);
+                                }
+                              }}
+                              tabIndex={0}
+                              role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)